apps/azure-devops/sbox/base/kustomization.yaml

```diff
@@ -6,4 +6,4 @@ resources:
   - azure-devops-token.yaml
   - ../../azure-devops-agent-keda/azure-devops-agent.yaml
 patchesStrategicMerge:
-  - ../../azure-devops-agent-keda/ptlsbox.yaml
\ No newline at end of file
+  - ../../azure-devops-agent-keda/sbox.yaml
```

Comment:
`../../azure-devops-agent-keda` references need moved to just one cluster, this case just 00